services/core-api/app/api/now_applications/resources/now_application_export_resource.py

```diff
@@ -424,6 +430,13 @@ def transform_data(obj):
                     ]
                     obj[key] = code_object[0].short_description if code_object and len(
                         code_object) > 0 else "N/A"
+                elif key in UNDERGROUND_EXPLORATION_CODE_FIELDS:
+                    type_codes = UndergroundExplorationType.get_all()
+                    code_object = [
+                        code for code in type_codes if code.underground_exploration_type_code == obj[key]
+                    ]
+                    obj[key] = code_object[0].description if code_object and len(
+                        code_object) > 0 else "N/A"
```

Comment:
Can you please make or use the existing constant, if there is one, for "N/A"?

Reply:
Updated!